lte/gateway/python/scripts/icmpv6.py

```diff
@@ -1,23 +1,47 @@
-""" Script to send icmpv6 data"""
-import sys
-import time
+#!/usr/bin/env python3
 
-from scapy.all import Ether, ICMPv6EchoRequest, IPv6, sendp
+"""
+Copyright 2022 The Magma Authors.
 
-dst = sys.argv[1]
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
 
-print("dst: %s" % dst)
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
 
+import binascii
+import socket
+import sys
+import time
 
-eth = Ether()
+from dpkt import ethernet, icmp6, ip, ip6
+from magma.pipelined.ifaces import get_mac_address_from_iface
 
-i = IPv6()
-i.dst = dst  # UE_IP
-i.src = "2001::2"
+dst = sys.argv[1]
 
-q = ICMPv6EchoRequest()
+print("dst: %s" % dst)
```

Comment:
Print for debugging?

Reply:
The print statement was already present before (see line 9). We could remove it.